documentación y manejo de errores de eliminar dirección

sofiavictoriafranco opened the pull request.

sofiavictoriafranco committed:
documentacion y manejo de errores de eliminar direccion

lechugaxthz reviewed (changes requested):
Fijate que no coinciden los mensajes de error entre `documentacion` (controller) y `manejo de error` (service)

sofiavictoriafranco committed:
modificacion de acuerdo a los comentarios

lechugaxthz reviewed (approved):
bien, ya para mas adelante aplicar el `@HttpCode(<tu valor de status>)` para poder personalizar los `status code` que devolvemos tras la petición

EsteChar2906 reviewed (changes requested):
en el archivo directions.service importas desde @nestjs/common en la linea 1 y 5

lechugaxthz committed:
Merge branch 'development' into 84-eliminar-direccion-manejo-de-errores-y-documentacion

lechugaxthz committed:
Update directions.service.ts

agregado de importaciones 
```ts 
  BadRequestException,
  InternalServerErrorException,
  NotFoundException,
```
dentro del @nestjs/common de linea 1

lechugaxthz committed:
Merge branch 'development' into 84-eliminar-direccion-manejo-de-errores-y-documentacion

Fab1chapulino reviewed (approved):
Parece estar todo bien, solo que en futuros commits hay que corregir los nombres de algunas dependencias, como en directions.controllers.ts o directions.service.ts, donde sus respectivas clases tienen 'Direetions...' en lugar de 'Directions...'. Lo mismo pasa con los Dto's, a excepción de eso todo está bien.

EsteChar2906 committed:
Merge branch 'development' into 84-eliminar-direccion-manejo-de-errores-y-documentacion
